### DISCUSSION TITLE
resize stream

### DISCUSSION THREAD

**Discussion by sander816** [datarhei/restreamer#538] [Q&A] [answered]
i have a rtsp stream running, at 1920x1080 resolution. Is it possible to resize the stream to 640x480 or sometinh similar?

**Comment by svenerbeck** [comment, marked as answer]
Hello @sander816 
Using the encoding feature, you can resize the stream to 640x480 or a similar resolution. Please check the video settings of your video stream, where you will find various solution presets and the option for custom settings to adjust the resolution as needed. Select the appropriate resolution and ensure that your desired aspect ratio is maintained.

Cheerio
Sven
